adjust_test.go

```diff
@@ -663,3 +663,90 @@ func TestCellNotOffsetButFormulaAffected(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "D3+D4", formula)
 }
+
+func TestCommonFormulaAdjustment(t *testing.T) {
+	f := NewFile()
+	// insert row in the middle of the range
+	assert.NoError(t, f.SetCellFormula("Sheet1", "B1", "=SUM(A2:A3)"))
```

Comment:
The formula functions may allow more than 2 arguments, the regular expression should be considered in this case. For example, the cell formula is `SUM(A2:A3:A4)`, the expected result is `SUM(A2:A4:A5)`, but now it was `SUM(A2:A4)`.